filebeat/files/filebeat.jinja

```diff
@@ -1,73 +1,27 @@
-{%- set filebeat = salt['pillar.get']('filebeat', {}) %}
+# Example configuration file
```

Comment:
You set your configuration in the pillar and it's then transported one-to-one to the filebeat config file. Filebeat uses YAML as well.